presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -383,6 +383,88 @@ or credentials for a specific use case (e.g., bucket/user specific credentials).
 This Hadoop configuration property must be set in the Hadoop configuration
 files referenced by the ``hive.config.resources`` Hive connector property.
 
+AWS Security Mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Presto supports flexible security mapping for AWS Lake Formation(and AWS Glue) and AWS S3 API calls, allowing for separate
+credentials or IAM roles for specific users.
+
+A security mapping entry can be of two types: ``s3`` or ``lakeformation``
+
+Each s3 security mapping entry may specify one match criteria. Available match criteria:
```

Comment:
```suggestion
Each ``s3`` security mapping entry may specify one match criteria. Available match criteria:
```
I thought about suggesting subheaders for s3 and lakeformation, but we're already at the bottom of the subsubheading formatting at the "AWS Security Mapping" heading. Suggest formatting ``s3`` and ``lakeformation`` as you did in line 392 to help the user find these as not-quite-headings.